tests/h/services/group_test.py

```diff
@@ -433,32 +483,50 @@ def test_provides_realtime_publisher_as_publish(self, patch, pyramid_request):
         })
 
 
-class GroupScopeWithOrigin(Matcher):
-    """Matches any GroupScope with the given origin."""
+@pytest.fixture
+def usr_svc(pyramid_request, db_session):
+    def fetch(userid):
+        # One doesn't want to couple to the user fetching service but
+        # we do want to be able to fetch user models for internal
+        # module behavior tests
+        return db_session.query(User).filter_by(userid=userid).one_or_none()
+    return fetch
 
-    def __init__(self, origin):
-        self.origin = origin
 
-    def __eq__(self, group_scope):
-        if not isinstance(group_scope, GroupScope):
-            return False
-        return group_scope.origin == self.origin
+@pytest.fixture
+def origins():
+    return ['http://example.com']
+
+
+@pytest.fixture
+def publish():
+    return mock.Mock()
```

Comment:
The `spec_set=[]` that was here was probably a good idea. It prevents typos like `m.asset_not_called()` from getting into the tests (and passing silently)